docs/AdminGuide.md

```diff
@@ -108,3 +108,54 @@ Example entry:
     "time": "2021-10-05 14:44:59",
 }
 ```
+
+## Update the expiry date of mismatches
+
+1. SSH into toolforge.
+
+    ```bash
+    ssh <your_username>@login.toolforge.org
+    ```
+
+1. The `become` commands allows you to sudo to another user while retaining your personalized environment.
+
+    ```bash
+    become mismatch-finder
+    ```
+
+1. Get into the database using the toolforge alias for accessing Mariadb 
+    ```bash
+    sql tools
+    ```
+
+1. Use the Wikidata mismatch finder database
+
+    ```bash
+    USE s54796__mismatch_finder; 
+    ```
+
```

Comment:
I'm not sure if the prefix (`s54796`) is same for every mismatch finder instance, so it'd be great to run `SHOW DATABASES;` command to be able to pick the correct DB.

Reply:
The database is indeed always the same. It just got that suffix number assigned automatically when it was created.

Reply:
The database will always be the same on Toolforge, but I think Hasan was thinking about other installations of Mismatch Finder? (E.g. our local installs.)

Reply:
It can be rewritten to `cd mismatch-finder-repo` and then `php artisan db` as @lucaswerkmeister mentions but I don't that's any better tbh. The db name is not changing. Toolforge just created this db since the start. This would be useful if there are plans to move the database at some point.